src/Grpc/HttpApi/src/Microsoft.AspNetCore.Grpc.Swagger/Microsoft.AspNetCore.Grpc.Swagger.csproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <Description>Swagger for gRPC ASP.NET Core</Description>
+    <PackageTags>gRPC RPC HTTP/2 REST Swagger OpenAPI</PackageTags>
+    <IsPackable>true</IsPackable>
+    <IsShipping>true</IsShipping>
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <TargetFramework>$(DefaultNetCoreTargetFramework)</TargetFramework>
+
+    <RestoreAdditionalProjectSources>$(RestoreAdditionalProjectSources);$(ArtifactsShippingPackagesDir)</RestoreAdditionalProjectSources>
+    <UseAspNetCoreSharedRuntime>true</UseAspNetCoreSharedRuntime>
```

Comment:
May be worth a comment in the projects (like this one) that reference neither Grpc.AspNetCore nor Grpc.AspNetCore.Server about how they use the shared Fx i.e. though src/Grpc/HttpApi/src/Microsoft.AspNetCore.Grpc.HttpApi/Microsoft.AspNetCore.Grpc.HttpApi.csproj